src/scenes/Login/Login.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import './_login.scss'
+import fetch from 'node-fetch'
+import PropTypes from 'prop-types'
+import { createBrowserHistory } from 'history'
+const history = createBrowserHistory()
+
+const Login = ({ setFoundUser }) => {
+  const [username, setUsername] = useState()
+  const [password, setPassword] = useState()
+  const [error, setError] = useState(null)
+  const [loading, setLoading] = useState(false)
+
+  const handleChangUsername = e => {
+    setUsername(e.target.value)
+  }
+  const handleChangePassword = e => {
+    setPassword(e.target.value)
+  }
+  const handleSubmit = async e => {
+    const loginUser = async () => {
+      return fetch(`http://localhost:3000/login?username=${username}&password=${password}`, {
+        method: 'GET',
+        headers: {
+          'content-type': 'application/json'
+        }
+      })
+        .then(response => response.json())
+    }
+    e.preventDefault()
+    const foundUser = await loginUser({
+      username,
+      password
+    })
+    setLoading(true)
+    setError(null)
+    if (setFoundUser(foundUser)) {
+      return history.replace({ pathname: '/' })
+    } else {
+      return <Login />
```

Comment:
Incorrect logic here
1. No need to return Login Component, you can choose to not return anything
2. if not found user, display an error message to let users know they are inputting the wrong username or password

Reply:
:> 
![image](https://user-images.githubusercontent.com/62316127/123357421-1b1d0d00-d52f-11eb-858a-fb79a9054216.png)